src/subdomains/generic/kyc/enums/review-status.enum.ts

```diff
@@ -0,0 +1,17 @@
+export enum ReviewStatus {
+  NOT_STARTED = 'NotStarted',
+  IN_PROGRESS = 'InProgress',
+  FINISHED = 'Finished',
+  EXTERNAL_REVIEW = 'ExternalReview',
+  INTERNAL_REVIEW = 'InternalReview',
+  MANUAL_REVIEW = 'ManualReview',
+  FAILED = 'Failed',
+  COMPLETED = 'Completed',
+  PARTIALLY_APPROVED = 'PartiallyApproved',
+  CANCELED = 'Canceled',
+  IGNORED = 'Ignored',
+  OUTDATED = 'Outdated',
+  DATA_REQUESTED = 'DataRequested',
+  PAUSED = 'Paused',
+  ON_HOLD = 'OnHold',
+}
```

Comment:
I'm not sure if it makes sense to combine the two enums into one. There are many status that are only used for KYC so it might be quite confusing for bank datas 🤔

Reply:
Haha, yes. As an outsider it's easier to see what's going on when using separate enums. But we can leave it as its is, if you prefer 😉